templates/kafka_logrotate.j2

```diff
@@ -0,0 +1,11 @@
+{{ kafka_log_files }} {
```

Comment:
I think it should be `{{ kafka_log_files }}/*.log { 
[...]
}`

Reply:
ok